src/main.rs

```diff
@@ -123,16 +123,16 @@ fn main() -> ! {
     });
 
     // Configure pin 25 for GPIO
-    p.PADS_BANK0.gpio25.write(|w| {
+    p.PADS_BANK0.gpio[25].write(|w| {
         // Output Disable off
         w.od().clear_bit();
         // Input Enable on
         w.ie().set_bit();
         w
     });
-    p.IO_BANK0.gpio25_ctrl.write(|w| {
+    p.IO_BANK0.gpio[25].gpio_ctrl.write(|w| {
         // Map pin 25 to SIO
-        w.funcsel().sio_25();
+        w.funcsel().sio_0();
```

Comment:
What is this change for?

Reply:
In that same commit (rp-rs/rp2040-pac#12), the enum with value 5 for FUNCSEL_A has been changed to SIO_0. What was sio_25 for pin 25 is now named sio_0. This is the same for all gpio pins.

https://github.com/rp-rs/rp2040-pac/blob/f4339cb856f082970767099a0079259f333cdc7e/src/io_bank0/gpio/gpio_ctrl.rs#L388